### PR TITLE
Fixed Shami.lua and added required items to item_basic.sql

### PR DESCRIPTION
**Pull request by Ninjistix**
added Kindred's Crests, High Kindred's Crests, Sacred Kindred's Crests trading cracked orbs and giving orbs

### CODE REVIEW

File: scripts/zones/Port_Jeuno/npcs/Shami.lua [hunk 1]
```diff
@@ -87,38 +87,89 @@ function onTrade(player,npc,trade)
...
             player:startEvent(0x000b); -- Themis Orb is ok, List where you can take the orb.
         end
--- TODO : Add in trade-in for cracked Kindred Crest orbs (and High Kindred Crest) and find cutscene that tells you where you can bring it        
+----------------------------------------------------------------------------------------------------------------------------
+-------- Trading Cracked KCNM Orbs or checking where you can bring the Orb -------------------------------------------------
+----------------------------------------------------------------------------------------------------------------------------
+    elseif (trade:hasItemQty(3351,1) and NumberItem == 1) then
+        if (player:getVar("PhobosOrbIsCracked") == 1) then
+            player:startEvent(0x0016); -- Phobos Orb is Cracked
+            player:setVar("PhobosOrbIsCracked",0);
```
Comment (xdemolish) [inline scripts/zones/Port_Jeuno/npcs/Shami.lua]: best to handle anything related to the end of a cutscene (such as quest progression) in onEventFinish

Comment (xdemolish) [inline scripts/zones/Port_Jeuno/npcs/Shami.lua]: actually, looks like you have and these setVars are just leftovers

Comment (Ninjistix) [inline scripts/zones/Port_Jeuno/npcs/Shami.lua]: that was the old line, just moved it to add the missing content (whoever was supposed to add)

**Comment by teschnei** [comment]
"No description provided."

the description is where you tell me what you actually fixed

**Comment by Ninjistix** [comment]
sorry updated description

File: scripts/zones/Port_Jeuno/npcs/Shami.lua (hunk 1 above)
Comment (xdemolish) [inline scripts/zones/Port_Jeuno/npcs/Shami.lua]: yep but you have this line here and in onEventFinish
only need it in onEventFinish (same with the other orb cracked vars, make sure they're set in onEventFinish only)

**Comment by Ninjistix** [comment]
branch got messed up had to reset

**Pull request closed without merging**